tests/testthat/test-remove-empties.R

```diff
@@ -122,3 +122,27 @@ test_that("remove_constant", {
     info="NA with other values is kept with na.rm"
   )
 })
+
+test_that("Messages are accurate with remove_empty and remove_constant", {
+  expect_message(
+    remove_empty(data.frame(A=NA, B=1), which="cols", quiet=FALSE),
+    regexp="Removing 1 empty columns of 2 columns total (Removed: A).",
+    fixed=TRUE
```

Comment:
I could see adding one test here where multiple columns are removed, to test how the delimiting of those names works.